Source/MaestroPlot.cpp

```diff
@@ -26,17 +43,36 @@ Maestro::WritePlotFile (const int step,
 
 	std::string plotfilename;
 
+    if (!is_small) {
+        plotfilename = plot_base_name;
+    } else {
+        plotfilename = small_plot_base_name;
+    }
+
 	if (step == 9999999) {
-		plotfilename = "plt_InitData";
+		if (plotfilename.back() == '_') {
+			plotfilename += "InitData";
+		} else {
+			plotfilename += + "_InitData";
+		}
+
 	}
 	else if (step == 9999998) {
```

Comment:
Currently, the plotfiles output during setup are all `plt_InitData`, `plt_after_DivuIter` etc, i.e. all with the prefix `plt` rather than the `plt_base_name` variable passed in by the user. I wanted to make sure that there was an underscore between the `plt_base_name` variable and the suffix, but didn't want there to be two underscores if the variable already had an underscore at the end (as is sometimes the case).